uvls/src/smt.rs

```diff
@@ -219,6 +230,22 @@ async fn find_fixed(
                     ))
                     .await?;
             }
+            SMTValueState::Core => {
```

Comment:
if both cases result in the same consequence/process you can use 
`SMTValueState::Core | SMTValueState::Core::FalseOptional => ...`

Reply:
At the moment I would say that this code can be removed because it is never called. You check if the ModuleSymbol in the SMTValueState::Core and SMTValueState::False Optional state is mapped, but the only case where a ModuleSymbol is mapped to one of these SMTValueStates is after lines 272 and 274, which are after the check. And this loop calls each ModuleSymbol only once.